frontend/src/views/Profile/Profile.tsx

```diff
@@ -68,20 +76,34 @@ export function Profile() {
             <div className={styles.contentContainer}>
                 {profile && <ProfileInfoPanel profile={profile} />}
                 <div className={styles.mainContent}>
-                    <div className={styles.settingsButtons}>
-                        <button type="button" className={styles.buttonOnCoverPhoto}>
-                            Edit cover photo
-                            <img
-                                className={styles.buttonIcon}
-                                src={imageIcon}
-                                alt="editCoverPhoto"
-                            />
-                        </button>
-                        <button type="button" className={styles.buttonOnCoverPhoto}>
-                            Edit profile
-                            <img className={styles.buttonIcon} src={editIcon} alt="editProfile" />
-                        </button>
-                    </div>
+                    {profileID === activeProfile.activeProfile?._id ? (
+                        <div className={styles.settingsButtons}>
+                            <button type="button" className={styles.buttonOnCoverPhoto}>
+                                Edit cover photo
+                                <img
+                                    className={styles.buttonIcon}
+                                    src={imageIcon}
+                                    alt="editCoverPhoto"
```

Comment:
suggestion: add `aria-hidden` attribute to images/icons that serve only visual purposes (button content/title should cover the context for this element)